tests/test_shipments.py

```diff
@@ -173,7 +173,7 @@ def test_add_tracking_data(self):
                     'speed': 34
                 },
                 'version': '1.0.0',
-                'device_id': 'adfc1e4c-7e61-4aee-b6f5-4d8b95a7ec75',
+                'device':  'adfc1e4c-7e61-4aee-b6f5-4d8b95a7ec75',
```

Comment:
This violates the tracking data JSON Schema. The field should be device_id, not device.